PluginsAndFeatures/azure-toolkit-for-intellij/src/com/microsoft/azure/hdinsight/spark/console/RunSparkScalaConsoleAction.kt

```diff
@@ -53,6 +53,8 @@ abstract class RunSparkScalaConsoleAction
     : AnAction(), RunConsoleAction.RunActionBase<LivySparkBatchJobRunConfigurationType>, ILogger {
     abstract val consoleRunConfigurationFactory: ScalaConsoleRunConfigurationFactory
 
+    abstract val selectMenuActionId: String
```

Comment:
Let the property not be a `doSth`

```suggestion
    abstract val selectedMenuActionId: String
```